server/routes/v1/profile.js

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+import { verifyUser } from '<middlewares>/verifyUser';
+import { update, getLoggedinProfile, getUserProfile } from '<controllers>/profile';
+import validateProfile from '<validations>/profile';
+import validateUuidParams from '<validations>';
+
+const router = express.Router();
+
+router.patch('/', verifyUser, validateProfile, update);
+router.get('/', verifyUser, getLoggedinProfile);
```

Comment:
Why do you have 3 routes ?

Reply:
The patch is to update the profile,
- the getLoggedinProfile returns the logged in user (picks the userId from the req object)
- the getUserProfile expects an id params

Reply:
and the 3rd route?